### PR TITLE
Trials review project

### PR DESCRIPTION
**Pull request by jdbs9514**
Make with:

- HTML
- CSS
- Linters

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        <br>
+        <br>
+        <div>Email</div>
+        <input type="text">
+        <br>
+        <br>
+        <div>Gender</div>
+        <input type="radio" id="gender" name="gender" value="male"/> Male
+        <input type="radio" id="gender" name="gender" value="female"/> Female 
+        <input type="radio" id="gender" name="gender" value="none"/> None
```
Comment (jdbs9514) [inline index.html:36]: good job! but Please kindly change the "id" to "class" because the "id" is only for ones unique tag

File: index.html [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        <br>
+        <br>
+        <div>How confiendent are you about javascript
+        </div>
+        <input type="radio" id="yy" name="yy" value="Beginner"/> Beginner
+        <input type="radio" id="yy" name="yy" value="Intermediate"/> Intermediate
+        <input type="radio" id="yy" name="yy" value="Advanced"/> Advanced 
+        <br>
+        <br>
+        <div>Your opinion on microverse
```
Comment (jdbs9514) [inline index.html:60]: please add "class" for your <div> tag

File: index.html [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+        <input type="radio" id="a" name="a" value="Very Good"/>Very Good
+        <input type="radio" id="a" name="a" value="Medium"/> Medium
+        <input type="radio" id="a" name="a" value="Bad"/> Bad
+        <br>
+        <br>
+        <div>On what scale do you rate microverse
+        </div>
+        <input type="radio" id="b" name="b" value="Very Good"/>Very Good
+        <input type="radio" id="b" name="b" value="Medium"/> Medium
+        <input type="radio" id="b" name="b" value="Bad"/> Bad
```
Comment (jdbs9514) [inline index.html:86]: please change the "id" to "class" because "id" is only for a unique tag.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,21 @@
+body {
+  text-align: center;
+  background-color: lightblue;
```
Comment (jdbs9514) [inline style.css:3]: please make sure that your background-color make changes of your project.

File: style.css [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+  /* background-image: url("m1.jfif"); */
+
+  /* background-color: aquamarine; */
```
Comment (jdbs9514) [inline style.css:7]: Good job! but Please kindly delete those irrelevant lines of your CSS.

File: style.css [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+}
+
+h1 {
+  font-family: 'Saira Condensed', sans-serif;
```
Comment (jdbs9514) [inline style.css:11]: please make sure that your font-family makes some changes to your code.

**Review by jdbs9514 — COMMENTED** [review]
# changes required ♻️ 

Please check my comments below and make all the required changes.
Submit the re-required when you´re done.

thanks 😁

File: style.css (hunk 5 above)
Comment (zieeco) [reply style.css:7]: **[REQUIRED CHANGES]**

- Hey, you've really done great with the review however, I think you could improve this one by re-evaluating your tone as you sound harsh and this is not ideal in making code reviews to foster good communication between you and the code author. Also, kindly add the reason why you want the lines of code to be removed. Technically, from the review, I think you meant "Unused code block", " which makes your codebase not clean and professional". Please, kindly adjust your tone and rephrase your sentence :pray:

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @jdbs9514 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  Nice beginning :+1: 
- Good use of github inline comments :fire: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danmainah — APPROVED** [review]
Hi @jdbs9514 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️  Good use of inline comments.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**